entity-types/ext-ftconnector/golden_metrics.yml

```diff
@@ -0,0 +1,18 @@
+syncstatus:
```

Comment:
We use camelcase for names: syncStatus, apply to other and in the sumary_metrics file

Reply:
fixed